modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/example.json

```diff
@@ -173,7 +173,7 @@
       },
       "servicePay": {
         "receivingMilitaryRetiredPay": "NO",
-        "futureMilitaryRetiredPay": "NO",
+        "futureMilitaryRetiredPay": "YES",
         "futureMilitaryRetiredPayExplanation": "Explanation of future military retired pay",
```

Comment:
This "bug" made its way in when the reorder happened.  These two values cannot be the same, so one needs to be 'YES' and the other 'NO' to pass validation so I just wanted to update one here so the example.json file could once again be used 'as is' to send a request hit.

Reply:
@RachalCassity  Apologies for missing this yesterday.  I see your point with this request.  Since this is a JSON file, and the file we use to display in our docs, I am trying to find a solution to for this.  Using a comment via ` // ` or something like that will invalidate the JSON (as far as I know) and we cannot add some kind of extra structure like `"_comment":`.  Is there anything else I have missed thinking of I could try here that would make you feel comfortable this has been addressed sufficiently?

Reply:
@RachalCassity Sure you are very busy, not trying to create more noise, just wanted to follow up on this question ^ .

Reply:
@RachalCassity I'm sure you are very busy, not trying to create more noise, just wanted to follow up on the question above since this has gotten approved, wanted to try to touch base on it, thanks!